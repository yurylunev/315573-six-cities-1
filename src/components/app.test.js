import React from "react";
import renderer from "react-test-renderer";
import App from "./app.jsx";

const testOffers = [
  {
    mark: `offerMark1`,
    imageURL: `imageURL1`,
    price: 1000,
    currency: `€`,
    rate: 100,
    name: `Name of offer 1`,
    type: `Apartment type 1`
  },
  {
    mark: `offerMark2`,
    imageURL: `imageURL2`,
    price: 2000,
    currency: `€`,
    rate: 200,
    name: `Name of offer 2`,
    type: `Apartment type 2`
  }
];
it(`App correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<App offers={testOffers}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

