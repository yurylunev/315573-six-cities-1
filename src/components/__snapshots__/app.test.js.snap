// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App correctly renders after relaunch 1`] = `
<h1>
  Loading ... 
</h1>
`;
